core/src/data/row.rs

```diff
@@ -111,4 +117,39 @@ impl Row {
 
         Ok(())
     }
+
+    pub fn to_rows(exprs_list: &[Vec<Expr>]) -> Result<Vec<Result<Self>>> {
+        let rows = exprs_list.iter().map(Row::try_from).collect::<Vec<_>>();
+
+        Ok(rows)
+    }
+
+    // fn to_row(exprs: &[Expr]) -> Result<Self> {
```

Comment:
Which is better to use between Associated Function `to_row` and below `try_from`?